### ISSUE TITLE
Is there any way I can get custom image album name list from iOS?

### ISSUE THREAD

**Issue by BROUDING** [facebook/react-native#10666]
`CameraRoll.getPhotos(fetchParams)
      .then((data) => this._appendImages(data, changeGallery), (e) => console.log(e));`

I used above code and as a result, in data there's a parameter called 'group_name'.
In Android, I can get all the album names.
In iOS, all the group_name is same as 'All Photos'.

So I can sort albums in Android, but not in iOS.

How can I get custom image album name list...?

p.s. Last time I asked, 'DanielMSchmidt' closed my issue without asnwering;;;
I need a comment from an author!
Thanks a lot.

**Comment by hramos** [comment]
The issue tracker is meant to be used to track bugs that need to be fixed. "How do I do X?" types of questions are best handled in Stack Overflow, which is a purpose-built Q&A forum. Would you like to [post this question to Stack Overflow with the tag #react-native](http://stackoverflow.com/questions/tagged/react-native)? 

As of this writing we have 789 open issues in the repository, so we need to be a bit more proactive about closing issues. I'm afraid I'll have to close this one out for now!

**Comment by BROUDING** [comment]
Okay I understood.
Thanks..

**Comment by DylanVann** [comment]
It seems like there's no way to do this with RN. We can get a list of images that are in albums:

```
CameraRoll.getPhotos({
  first: 10,
  groupTypes: 'Album',
})
```

Which is the first 10 images that are in albums, not the first image in the first 10 albums (which would be much more convenient). But we can't get a list of albums.

Seems like a feature request.

**Comment by hramos** [comment]
Feel free to post it to Product Pains as a feature request: https://productpains.com/product/react-native/ - it's been helpful when it comes to prioritizing what people work on next.

**Comment by BROUDING** [comment]
@DylanVann , @hramos 

Gracias gentlemen...
for Android, I fixed CameraRoll as my taste and getting album list pretty well.
But since I strongly think this feature will be used a lot, I'll request on there for potential better work !

Have a good day !

**Comment by kytwb** [comment]
@DylanVann @BROUDING did you guys find a workaround for this? I managed to get a list of albums using this module https://github.com/shimohq/react-native-albums `getAlbumList` method but the `getImageList` isn't implemented in the iOS version. I then tried to use the built-in `CameraRoll.getPhotos` with `groupName` set to the previously retrieved album name, but it doesn't match with what's acceptable:

![simulator screen shot - iphone 6 - 2018-02-13 at 16 03 45](https://user-images.githubusercontent.com/412895/36141534-a7764d72-10d7-11e8-9d62-aaaa51fb8690.png)

Just leaving this here as that's as far as I dug it for now.
